generators/client-2/templates/src/main/webapp/swagger-ui/_index.html

```diff
@@ -5,23 +5,25 @@
     <title>Swagger UI</title>
     <link rel="icon" type="image/png" href="images/favicon-32x32.png" sizes="32x32" />
     <link rel="icon" type="image/png" href="images/favicon-16x16.png" sizes="16x16" />
-    <link href='../bower_components/swagger-ui/dist/css/typography.css' media='screen' rel='stylesheet' type='text/css'/>
-    <link href='../bower_components/swagger-ui/dist/css/reset.css' media='screen' rel='stylesheet' type='text/css'/>
-    <link href='../bower_components/swagger-ui/dist/css/screen.css' media='screen' rel='stylesheet' type='text/css'/>
-    <link href='../bower_components/swagger-ui/dist/css/reset.css' media='print' rel='stylesheet' type='text/css'/>
-    <link href='../bower_components/swagger-ui/dist/css/print.css' media='print' rel='stylesheet' type='text/css'/>
-    <script src='../bower_components/swagger-ui/dist/lib/jquery-1.8.0.min.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/jquery.slideto.min.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/jquery.wiggle.min.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/jquery.ba-bbq.min.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/handlebars-2.0.0.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/underscore-min.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/backbone-min.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/swagger-ui.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/highlight.7.3.pack.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/jsoneditor.min.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/marked.js' type='text/javascript'></script>
-    <script src='../bower_components/swagger-ui/dist/lib/swagger-oauth.js' type='text/javascript'></script>
+    <link href='../swagger-lib/dist/css/typography.css' media='screen' rel='stylesheet' type='text/css'/>
```

Comment:
can we put these in the same `swagger-ui`  folder instead of creating a new folder? and where are the files btw?